### ISSUE TITLE
Set coordinates for OpenWeatherMap data manually

### ISSUE THREAD

**Issue by jeroenterheerdt** [jeroenterheerdt/HAsmartirrigation#351]
### Discussed in https://github.com/jeroenterheerdt/HAsmartirrigation/discussions/346

<div type='discussions-op-text'>

<sup>Originally posted by **maehmann** April 13, 2024</sup>
Hey,

I'm trying to set up your great looking integration for my situation. This is where it get's a bit tricky. I have two locations separated by 70km that run on one HA installation being linked through a VPN. So my Raspi sits in location A but I want to setup the irrigation system for location B which is not configure as the main location in HA obviously.

On location B I do not have a precipitation sensor. Only temp, pressure, humidity, due point. If I understand correctly then all info missing from the sensors will be taken from OpenWeatherMap depending on the set coordinates in HA.

Is this correct? And if so, is there a way to set a new location manually somehow (by coordinates or by a defined zone in HA)?

Thank you for your hard work and in advance for your reply!

Christoph</div>

**Comment by motogon** [comment]
I agree, the ability to specify coordinates in the module itself is very useful.

**Comment by jeroenterheerdt** [comment]
Being worked on in #623

**Comment by altmenorg** [comment]
Manual coordinates are now supported: enable them in the integration config and set latitude, longitude and elevation, used instead of the Home Assistant location. Available in v2026.7.0-beta0. Closing as implemented.
